### PR TITLE
fix typo in filename and redirect

### PR DESCRIPTION
**Pull request by mamccrea**
## Summary

There was a small typo in the file name.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -4193,6 +4193,10 @@
     {
       "source_path": "docs/visual-basic/reference/vb6-support.md",
       "redirect_url": "https://docs.microsoft.com/previous-versions/visualstudio/visual-basic-6/visual-basic-6-support-policy"
+    },
+    {
+      "source_path": "docs/spark/tutorials/ml-sentment-analysis.md",
```
Comment (pkulikov) [inline .openpublishing.redirection.json]: We try to keep entries in this file sorted alphabetically by `source_path`

Comment (BillWagner) [reply .openpublishing.redirection.json]: Background: We started keeping this sorted because it has worked well to avoid any merge conflicts in this file.

Comment (mamccrea) [reply .openpublishing.redirection.json]: Thanks - this makes sense. I didn't realize they were sorted alphabetically.

File: .openpublishing.redirection.json [hunk 2]
```diff
@@ -3372,6 +3372,10 @@
       "source_path": "docs/scenarios/solution-authoring/target-dotnetcore-with-msbuild.md",
       "redirect_url": "/dotnet/core/tools/cli-msbuild-architecture"
     },
+    {
+      "source_path": "docs/spark/tutorials/ml-sentment-analysis.md",
+      "redirect_url": "dotnet/spark/tutorials/ml-sentiment-analysis"
```
Comment (gewarren) [inline .openpublishing.redirection.json]:
```suggestion
      "redirect_url": "/dotnet/spark/tutorials/ml-sentiment-analysis"
```

**Comment by gewarren** [comment]
As a tip, you can check if the redirect is working properly by clicking this:

image

**Comment by gewarren** [comment]
It's still not working - for some reason it's redirecting to this, which is wrong because dotnet/spark/tutorials is repeated, but the redirection file entry looks fine to me!

https://review.docs.microsoft.com/en-us/dotnet/spark/tutorials/**dotnet/spark/tutorials**/ml-sentiment-analysis?branch=pr-en-us-17626

**Comment by mamccrea** [comment]
Hmm... it' appears to be working for me. This would be the link to be redirected, and it redirects correctly for me:

https://review.docs.microsoft.com/en-us/dotnet/spark/tutorials/ml-sentment-analysis?branch=pr-en-us-17626

**Comment by gewarren** [comment]
> Hmm... it' appears to be working for me. This would be the link to be redirected, and it redirects correctly for me:
> 
> https://review.docs.microsoft.com/en-us/dotnet/spark/tutorials/ml-sentment-analysis?branch=pr-en-us-17626

Okay, working for me now too. I'll approve this.

**Review by gewarren — APPROVED** [review]

**Pull request merged**